dali/imgcodec/util/convert_gpu.cu

```diff
@@ -45,10 +45,24 @@ void LaunchSliceFlipNormalizePermutePad(
   args_container.emplace_back(out_shape, in_shape);
   auto &args = args_container[0];
 
+  bool swap_xy = orientation.rotate % 180 == 90;
```

Comment:
How about making this a methods of the Orientation struct?

Reply:
```
  bool DoSwapXYConsideringRotation() {
    return rotate % 180 == 90;
  }

  bool DoFlipXConsideringRotation() {
    return (rotate == 90 || rotate == 180) ^ flip_x;
  }

  bool DoFlipYConsideringRotation() {
    return (rotate == 180 || rotate == 270) ^ flip_y;
  }
```

What do you think about these? I don't really like the names, but don't have any better idea.

Reply:
If you add those, then we should probably use them in Convert CPU